client/src/Visualizations/LineChart/LineChart.jsx

```diff
@@ -200,7 +230,7 @@ class LineGraph extends React.Component {
   getDataYScale() {
     return d3
       .scaleLinear()
-      .domain([1500, 2500])
+      .domain([0, 2500])
```

Comment:
This is fine for now. I'll have to find the min and max of the countries to populate later though